src/js/components/TeamDetail.js

```diff
@@ -0,0 +1,230 @@
+import Api from "../api/api.js";
+import {$, $$} from "../utils/querySelector.js";
+
+export default function teamDetail() {
+  this.teamId = "";
+
+  const setMemberList = async () => {
+    addMembers(await getTeamMembers());
+  }
+
+  const getTeamMembers = async () => {
+    const params = new URLSearchParams(window.location.search);
+    this.teamId = params.get("id");
+    const data = await Api.getFetch(`/api/teams/${this.teamId}`);
+
+    $("#user-title").querySelector("strong").innerHTML = data.name;
+
+    return data.members;
+  };
+
+  const addEvent = () => {
+    const $listContainer = $(".todoapp-list-container");
+
+    $listContainer.addEventListener('click', ({target, target: {value, classList}}) => {
+
+      if (classList.contains('destroy')) deleteItem(target)
+      else if (classList.contains('toggle')) completeItem(target)
+      else if (classList.contains('clear-completed')) deleteAll(target)
+      else if (classList.contains('filter')) filterList(target)
+    });
+
+    $listContainer.addEventListener('change', ({target, target: {value, classList}}) => {
+      if (classList.contains('chip')) changePriority(target, value);
+    });
+
+    $listContainer.addEventListener('dblclick', ({target, target: {value, classList}}) => {
+      if (classList.contains('label')) editItem(target);
+    });
+
+    $listContainer.addEventListener('keyup', ({target, target: {value, classList}, key}) => {
+      key === "Enter" && addTodo(target);
+    });
+
+    $('#add-user-button').addEventListener('click', async () => {
+      const result = prompt('새로운 팀원 이름을 입력해주세요');
```

Comment:
아닐 수도 있지만, prompt 실행 후 취소 버튼 누르면 result에 undefined 또는 null 값이 들어갈 거 같은데
그럼 그때 (취소 버튼 눌렀을 때) 불 필요한 작업 postFetch 요청과 setMemberList 요청이 발생할 거 같은데 방어 코드 작성해주시는 거 어떤가요??

```suggestion
      const result = prompt('새로운 팀원 이름을 입력해주세요');
      if (!result) return;
```